[BUG] creating ovh_cloud_project_kube_oidc resource fails

### Describe the bug

When creating the `ovh_cloud_project_kube_oidc` resource, terraform throws the following error:

```
module.cluster.ovh_cloud_project_kube_oidc.oidc: Creating...
╷
│ Error: timeout while waiting kube 019adbb1-cfa5-76ab-9a00-ee2d784500d9 to be READY: unexpected state 'INSTALLING', wanted target 'READY'. last error: %!s(<nil>)
│ 
│   with module.cluster.ovh_cloud_project_kube_oidc.oidc,
│   on ../../modules/ovh/cluster/cluster.tf line 95, in resource "ovh_cloud_project_kube_oidc" "oidc":
│   95: resource "ovh_cloud_project_kube_oidc" "oidc" {
│ 
╵
```

### Terraform Version

1.14.0

### OVH Terraform Provider Version

2.10.0

### Affected Resource(s)

- ovh_cloud_project_kube_oidc


### Terraform Configuration Files

```hcl
data "ovh_order_cart" "cart" {
  ovh_subsidiary = var.ovh_subsidiary
}

data "ovh_order_cart_product_plan" "cloud" {
  cart_id        = data.ovh_order_cart.cart.id
  price_capacity = "renew"
  product        = "cloud"
  plan_code      = "project.2018"
}

resource "ovh_cloud_project" "project" {
  ovh_subsidiary = data.ovh_order_cart.cart.ovh_subsidiary
  description    = "${var.instance}-${var.cluster}"

  plan {
    duration     = data.ovh_order_cart_product_plan.cloud.selected_price[0].duration
    plan_code    = data.ovh_order_cart_product_plan.cloud.plan_code
    pricing_mode = data.ovh_order_cart_product_plan.cloud.selected_price[0].pricing_mode
  }
}

locals {
  project_id = ovh_cloud_project.project.project_id
}

# Look up the vRack attached to the cloud project
data "ovh_cloud_project_vrack" "vrack" {
  service_name = local.project_id
}

# Private network for cluster nodes
resource "ovh_cloud_project_network_private" "cluster_network" {
  service_name = local.project_id
  vlan_id      = var.vlan_id
  name         = "${var.instance}-${var.cluster}-network"
  regions      = [var.ovh_region]

  depends_on = [data.ovh_cloud_project_vrack.vrack]
}

resource "ovh_cloud_project_network_private_subnet" "cluster_subnet" {
  service_name = local.project_id
  network_id   = ovh_cloud_project_network_private.cluster_network.id
  region       = var.ovh_region
  start        = cidrhost(var.private_network_cidr, 100)
  end          = cidrhost(var.private_network_cidr, 200)
  network      = var.private_network_cidr
  dhcp         = true
  no_gateway   = false
}

resource "ovh_cloud_project_gateway" "cluster_gateway" {
  service_name = local.project_id
  name         = "${var.instance}-${var.cluster}-gateway"
  model        = var.gateway_model
  region       = var.ovh_region
  network_id   = tolist(ovh_cloud_project_network_private.cluster_network.regions_attributes[*].openstackid)[0]
  subnet_id    = ovh_cloud_project_network_private_subnet.cluster_subnet.id
}

# OVH Managed Kubernetes cluster
resource "ovh_cloud_project_kube" "cluster" {
  service_name       = local.project_id
  name               = "${var.instance}-${var.cluster}"
  region             = var.ovh_region
  version            = var.kubernetes_version
  private_network_id = tolist(ovh_cloud_project_network_private.cluster_network.regions_attributes[*].openstackid)[0]
  nodes_subnet_id    = ovh_cloud_project_network_private_subnet.cluster_subnet.id
  plan               = "standard"

  depends_on = [ovh_cloud_project_gateway.cluster_gateway]
}

# Node pools - one per availability zone for multi-AZ deployment
resource "ovh_cloud_project_kube_nodepool" "pool" {
  for_each = toset(var.node_pool_zones)

  service_name       = local.project_id
  kube_id            = ovh_cloud_project_kube.cluster.id
  name               = "pool-${var.instance}-${var.cluster}-${replace(each.key, "-", "")}"
  flavor_name        = var.node_machine_type
  desired_nodes      = var.node_min_amount_per_pool
  min_nodes          = var.node_min_amount_per_pool
  max_nodes          = var.node_max_amount_per_pool
  autoscale          = true
  availability_zones = [each.key]
}

# OIDC configuration
# Note: This resource modifies the cluster and triggers a reconfiguration
# The OVH provider has a bug where it doesn't wait long enough for the cluster
# to return to READY state after applying OIDC configuration
resource "ovh_cloud_project_kube_oidc" "oidc" {
  service_name         = local.project_id
  kube_id              = ovh_cloud_project_kube.cluster.id
  client_id            = var.cluster_oidc_client_id
  issuer_url           = "https://${var.cluster_sso_server}"
  oidc_username_claim  = var.cluster_oidc_username_claim
  oidc_groups_claim    = [var.cluster_oidc_groups_claim]

  timeouts {
    create = "30m"
    update = "30m"
    delete = "10m"
  }

  depends_on = [ovh_cloud_project_kube_nodepool.pool]
}
```

### Debug Output

I have not provided the full debug output as that contains credentials.

<details>
<summary>relevant debug logs of OIDC provider creation</summary>

```
2025-12-01T22:58:22.479+0100 [DEBUG] module.cluster.ovh_cloud_project_kube_oidc.oidc: applying the planned Create change
2025-12-01T22:58:22.479+0100 [TRACE] GRPCProvider.v6: ApplyResourceChange
2025-12-01T22:58:22.479+0100 [TRACE] GRPCProvider.v6: returning cached schema: EXTRA_VALUE_AT_END=registry.terraform.io/ovh/ovh
2025-12-01T22:58:22.480+0100 [TRACE] provider.terraform-provider-ovh_v2.10.0: Received request: tf_resource_type=ovh_cloud_project_kube_oidc tf_rpc=ApplyResourceChange tf_provider_addr=registry.terraform.io/ovh/ovh @caller=github.com/hashicorp/terraform-plugin-go@v0.26.0/tfprotov6/tf6server/server.go:853 @module=sdk.proto tf_proto_version=6.8 tf_req_id=b32225cb-d0ff-9f0d-b38a-f4be6b14386b timestamp="2025-12-01T22:58:22.480+0100"
2025-12-01T22:58:22.480+0100 [TRACE] provider.terraform-provider-ovh_v2.10.0: Sending request downstream: @caller=github.com/hashicorp/terraform-plugin-go@v0.26.0/tfprotov6/internal/tf6serverlogging/downstream_request.go:22 @module=sdk.proto tf_proto_version=6.8 tf_provider_addr=registry.terraform.io/ovh/ovh tf_req_id=b32225cb-d0ff-9f0d-b38a-f4be6b14386b tf_resource_type=ovh_cloud_project_kube_oidc tf_rpc=ApplyResourceChange timestamp="2025-12-01T22:58:22.480+0100"
2025-12-01T22:58:22.480+0100 [TRACE] provider.terraform-provider-ovh_v2.10.0: calling downstream server: tf_mux_provider=tf5to6server.v5tov6Server tf_rpc=ApplyResourceChange @caller=github.com/hashicorp/terraform-plugin-mux@v0.18.0/internal/logging/mux.go:19 @module=sdk.mux timestamp="2025-12-01T22:58:22.480+0100"
2025-12-01T22:58:22.480+0100 [TRACE] provider.terraform-provider-ovh_v2.10.0: Calling downstream: tf_mux_provider=tf5to6server.v5tov6Server tf_provider_addr=registry.terraform.io/ovh/ovh tf_req_id=b32225cb-d0ff-9f0d-b38a-f4be6b14386b tf_resource_type=ovh_cloud_project_kube_oidc @caller=github.com/hashicorp/terraform-plugin-sdk/v2@v2.36.1/helper/schema/resource.go:968 @module=sdk.helper_schema tf_rpc=ApplyResourceChange timestamp="2025-12-01T22:58:22.480+0100"
2025-12-01T22:58:22.480+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: 2025/12/01 22:58:22 [DEBUG] Will create OIDC: &{ClientID:kubernetes.eidp.io IssuerUrl:https://login.eidp.io/realms/eidp UsernameClaim:email UsernamePrefix: GroupsClaim:[roles] GroupsPrefix: RequiredClaim:[] SigningAlgs:[] CaContent:}
2025-12-01T22:58:22.480+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: 2025/12/01 22:58:22 [DEBUG] OVH API Request Details:
2025-12-01T22:58:22.480+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: ---[ REQUEST ]---------------------------------------
2025-12-01T22:58:22.480+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: POST /1.0/cloud/project/b1fe49d3b20440888b5d8c73125c90cd/kube/019adbb1-cfa5-76ab-9a00-ee2d784500d9/openIdConnect HTTP/1.1
2025-12-01T22:58:22.480+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: Host: eu.api.ovh.com
2025-12-01T22:58:22.480+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: User-Agent: github.com/ovh/go-ovh (Terraform/2.10.0/78325622b7eb6b04792b57d37cf8b2b1dc7efb2e)
2025-12-01T22:58:22.480+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: Content-Length: 224
2025-12-01T22:58:22.480+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: Accept: application/json
2025-12-01T22:58:22.480+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: Authorization: Bearer eyJhbGciOiJFZERTQSIsImtpZCI6IkVGNThFMkUxMTFBODNCREFEMDE4OUUzMzZERTk3MDhFNjRDMDA4MDEiLCJraW5kIjoib2F1dGgyIiwidHlwIjoiSldUIn0.eyJBY2Nlc3NUb2tlbiI6IjgyMWE0NTQ5MjYxYTIyNjFkMmY1NGFlYjBlNmUxNTVlN2JhMmQyNzhkNTc3NTJjZTRkMzM1NmUzN2IyZmY4MTAiLCJpYXQiOjE3NjQ2MjYzMDJ9.IeIMfaerZ_hvbApP_2upmUhoEddw5cK3hcWYyuU1rDm8xjy-8QBodTOPecnpa_rPKsR6u1svGM-UF0MurNOMAg
2025-12-01T22:58:22.480+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: Content-Type: application/json;charset=utf-8
2025-12-01T22:58:22.480+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: Accept-Encoding: gzip
2025-12-01T22:58:22.480+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0
2025-12-01T22:58:22.480+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: {
2025-12-01T22:58:22.480+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:  "clientId": "kubernetes.eidp.io",
2025-12-01T22:58:22.480+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:  "issuerUrl": "https://login.eidp.io/realms/eidp",
2025-12-01T22:58:22.480+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:  "usernameClaim": "email",
2025-12-01T22:58:22.480+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:  "usernamePrefix": "",
2025-12-01T22:58:22.480+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:  "groupsClaim": [
2025-12-01T22:58:22.480+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:   "roles"
2025-12-01T22:58:22.480+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:  ],
2025-12-01T22:58:22.480+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:  "groupsPrefix": "",
2025-12-01T22:58:22.480+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:  "requiredClaim": [],
2025-12-01T22:58:22.480+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:  "signingAlgorithms": [],
2025-12-01T22:58:22.480+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:  "caContent": ""
2025-12-01T22:58:22.480+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: }
2025-12-01T22:58:22.480+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: -----------------------------------------------------
2025-12-01T22:58:22.714+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: 2025/12/01 22:58:22 [DEBUG] OVH API Response Details:
2025-12-01T22:58:22.714+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: ---[ RESPONSE ]--------------------------------------
2025-12-01T22:58:22.714+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: HTTP/2.0 200 OK
2025-12-01T22:58:22.714+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: Access-Control-Allow-Headers: X-Ovh-Timestamp, X-Ovh-Consumer, X-Ovh-Application, X-Ovh-Signature, X-Ovh-Session, Authorization, Content-Type, X-Challenge-Response, X-Challenge-Payload, User-Agent
2025-12-01T22:58:22.714+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: Access-Control-Allow-Methods: DELETE, GET, POST, PUT
2025-12-01T22:58:22.714+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: Access-Control-Allow-Origin: *
2025-12-01T22:58:22.714+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: Access-Control-Expose-Headers: X-Pagination-Cursor-Next, X-Ovh-Queryid
2025-12-01T22:58:22.714+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: Cache-Control: no-cache, no-store
2025-12-01T22:58:22.714+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: Content-Type: application/json; charset=utf-8
2025-12-01T22:58:22.714+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: Date: Mon, 01 Dec 2025 21:58:22 GMT
2025-12-01T22:58:22.714+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: Server: nginx
2025-12-01T22:58:22.714+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: Vary: Accept-Encoding
2025-12-01T22:58:22.714+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: X-Content-Type-Options: nosniff
2025-12-01T22:58:22.714+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: X-Frame-Options: DENY
2025-12-01T22:58:22.714+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: X-Iplb-Instance: 47073
2025-12-01T22:58:22.714+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: X-Iplb-Request-Id: D50A8CEF:E855_8D5FBADF:01BB_692E0F7E_D14ED4E:16C86
2025-12-01T22:58:22.714+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: X-Ovh-Queryid: EU.ext-4.692e0f7e.1273594.69dffee7d9f6e76af38b9b9b09241b91
2025-12-01T22:58:22.714+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: X-Xss-Protection: 1; mode=block
2025-12-01T22:58:22.714+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0
2025-12-01T22:58:22.714+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: {
2025-12-01T22:58:22.714+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:  "issuerUrl": "https://login.eidp.io/realms/eidp",
2025-12-01T22:58:22.714+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:  "clientId": "kubernetes.eidp.io",
2025-12-01T22:58:22.714+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:  "usernameClaim": "email",
2025-12-01T22:58:22.714+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:  "usernamePrefix": "",
2025-12-01T22:58:22.714+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:  "groupsClaim": [
2025-12-01T22:58:22.714+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:   "roles"
2025-12-01T22:58:22.714+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:  ],
2025-12-01T22:58:22.714+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:  "groupsPrefix": "",
2025-12-01T22:58:22.714+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:  "signingAlgorithms": null,
2025-12-01T22:58:22.714+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:  "caContent": "",
2025-12-01T22:58:22.714+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:  "requiredClaim": null
2025-12-01T22:58:22.714+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: }
2025-12-01T22:58:22.714+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: -----------------------------------------------------
2025-12-01T22:58:22.714+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: 2025/12/01 22:58:22 [DEBUG] Waiting for kube 019adbb1-cfa5-76ab-9a00-ee2d784500d9 to be READY
2025-12-01T22:58:22.714+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: 2025/12/01 22:58:22 [DEBUG] Waiting for state to become: [READY]
2025-12-01T22:58:27.023+0100 [TRACE] dag/walk: vertex "root" is waiting for "module.cluster.provider[\"registry.terraform.io/ovh/ovh\"] (close)"
2025-12-01T22:58:27.023+0100 [TRACE] dag/walk: vertex "module.cluster.provider[\"registry.terraform.io/ovh/ovh\"] (close)" is waiting for "module.cluster.ovh_cloud_project_kube_oidc.oidc"
2025-12-01T22:58:27.468+0100 [TRACE] dag/walk: vertex "module.cluster (close)" is waiting for "module.cluster.ovh_cloud_project_kube_oidc.oidc"
2025-12-01T22:58:27.715+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: 2025/12/01 22:58:27 [DEBUG] OVH API Request Details:
2025-12-01T22:58:27.716+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: ---[ REQUEST ]---------------------------------------
2025-12-01T22:58:27.716+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: GET /1.0/cloud/project/b1fe49d3b20440888b5d8c73125c90cd/kube/019adbb1-cfa5-76ab-9a00-ee2d784500d9 HTTP/1.1
2025-12-01T22:58:27.716+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: Host: eu.api.ovh.com
2025-12-01T22:58:27.716+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: User-Agent: github.com/ovh/go-ovh (Terraform/2.10.0/78325622b7eb6b04792b57d37cf8b2b1dc7efb2e)
2025-12-01T22:58:27.716+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: Accept: application/json
2025-12-01T22:58:27.716+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: Authorization: Bearer eyJhbGciOiJFZERTQSIsImtpZCI6IkVGNThFMkUxMTFBODNCREFEMDE4OUUzMzZERTk3MDhFNjRDMDA4MDEiLCJraW5kIjoib2F1dGgyIiwidHlwIjoiSldUIn0.eyJBY2Nlc3NUb2tlbiI6IjgyMWE0NTQ5MjYxYTIyNjFkMmY1NGFlYjBlNmUxNTVlN2JhMmQyNzhkNTc3NTJjZTRkMzM1NmUzN2IyZmY4MTAiLCJpYXQiOjE3NjQ2MjYzMDJ9.IeIMfaerZ_hvbApP_2upmUhoEddw5cK3hcWYyuU1rDm8xjy-8QBodTOPecnpa_rPKsR6u1svGM-UF0MurNOMAg
2025-12-01T22:58:27.716+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: Accept-Encoding: gzip
2025-12-01T22:58:27.716+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0
2025-12-01T22:58:27.716+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0
2025-12-01T22:58:27.716+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: -----------------------------------------------------
2025-12-01T22:58:27.808+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: 2025/12/01 22:58:27 [DEBUG] OVH API Response Details:
2025-12-01T22:58:27.808+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: ---[ RESPONSE ]--------------------------------------
2025-12-01T22:58:27.808+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: HTTP/2.0 200 OK
2025-12-01T22:58:27.808+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: Access-Control-Allow-Headers: X-Ovh-Timestamp, X-Ovh-Consumer, X-Ovh-Application, X-Ovh-Signature, X-Ovh-Session, Authorization, Content-Type, X-Challenge-Response, X-Challenge-Payload, User-Agent
2025-12-01T22:58:27.808+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: Access-Control-Allow-Methods: DELETE, GET, PUT
2025-12-01T22:58:27.808+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: Access-Control-Allow-Origin: *
2025-12-01T22:58:27.808+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: Access-Control-Expose-Headers: X-Pagination-Cursor-Next, X-Ovh-Queryid
2025-12-01T22:58:27.808+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: Cache-Control: no-cache, no-store
2025-12-01T22:58:27.808+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: Content-Type: application/json; charset=utf-8
2025-12-01T22:58:27.808+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: Date: Mon, 01 Dec 2025 21:58:27 GMT
2025-12-01T22:58:27.808+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: Server: nginx
2025-12-01T22:58:27.808+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: Vary: Accept-Encoding
2025-12-01T22:58:27.808+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: X-Content-Type-Options: nosniff
2025-12-01T22:58:27.808+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: X-Frame-Options: DENY
2025-12-01T22:58:27.808+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: X-Iplb-Instance: 47058
2025-12-01T22:58:27.808+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: X-Iplb-Request-Id: D50A8CEF:E857_8D5FBADF:01BB_692E0F83_CF42A76:2BB40
2025-12-01T22:58:27.808+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: X-Ovh-Queryid: EU.ext-3.692e0f83.1440590.5f9c8c63179eba9129e360f793b9bfed
2025-12-01T22:58:27.808+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: X-Xss-Protection: 1; mode=block
2025-12-01T22:58:27.809+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0
2025-12-01T22:58:27.809+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: {
2025-12-01T22:58:27.809+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:  "id": "019adbb1-cfa5-76ab-9a00-ee2d784500d9",
2025-12-01T22:58:27.809+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:  "region": "EU-WEST-PAR",
2025-12-01T22:58:27.809+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:  "name": "eidp-ovh-demo",
2025-12-01T22:58:27.809+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:  "url": "https://0uk1rc5x.c1.eu-west-par.k8s.ovh.net",
2025-12-01T22:58:27.809+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:  "nodesUrl": "N/A",
2025-12-01T22:58:27.809+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:  "version": "1.34.1-1",
2025-12-01T22:58:27.809+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:  "nextUpgradeVersions": [],
2025-12-01T22:58:27.809+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:  "plan": "standard",
2025-12-01T22:58:27.809+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:  "kubeProxyMode": "ipvs",
2025-12-01T22:58:27.809+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:  "customization": {
2025-12-01T22:58:27.809+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:   "apiServer": {
2025-12-01T22:58:27.809+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:    "admissionPlugins": {
2025-12-01T22:58:27.809+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:     "enabled": [
2025-12-01T22:58:27.809+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:      "NodeRestriction",
2025-12-01T22:58:27.809+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:      "AlwaysPullImages"
2025-12-01T22:58:27.809+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:     ]
2025-12-01T22:58:27.809+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:    }
2025-12-01T22:58:27.809+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:   }
2025-12-01T22:58:27.809+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:  },
2025-12-01T22:58:27.809+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:  "status": "INSTALLING",
2025-12-01T22:58:27.809+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:  "updatePolicy": "ALWAYS_UPDATE",
2025-12-01T22:58:27.809+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:  "isUpToDate": true,
2025-12-01T22:58:27.809+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:  "controlPlaneIsUpToDate": true,
2025-12-01T22:58:27.809+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:  "privateNetworkId": "125d3959-5f94-4407-804d-06c6d8660175",
2025-12-01T22:58:27.809+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:  "nodesSubnetId": "889d8c65-4d62-4f48-8f1f-a24987cee8a7",
2025-12-01T22:58:27.809+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:  "privateNetworkConfiguration": {
2025-12-01T22:58:27.809+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:   "privateNetworkRoutingAsDefault": true,
2025-12-01T22:58:27.809+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:   "defaultVrackGateway": ""
2025-12-01T22:58:27.809+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:  },
2025-12-01T22:58:27.809+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:  "createdAt": "2025-12-01T20:54:17Z",
2025-12-01T22:58:27.809+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:  "updatedAt": "2025-12-01T20:54:17Z",
2025-12-01T22:58:27.809+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0:  "auditLogsSubscribed": false
2025-12-01T22:58:27.809+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: }
2025-12-01T22:58:27.809+0100 [DEBUG] provider.terraform-provider-ovh_v2.10.0: -----------------------------------------------------
2025-12-01T22:58:27.809+0100 [TRACE] provider.terraform-provider-ovh_v2.10.0: Called downstream: @caller=github.com/hashicorp/terraform-plugin-sdk/v2@v2.36.1/helper/schema/resource.go:970 tf_mux_provider=tf5to6server.v5tov6Server tf_provider_addr=registry.terraform.io/ovh/ovh tf_req_id=b32225cb-d0ff-9f0d-b38a-f4be6b14386b tf_resource_type=ovh_cloud_project_kube_oidc @module=sdk.helper_schema tf_rpc=ApplyResourceChange timestamp="2025-12-01T22:58:27.808+0100"
2025-12-01T22:58:27.811+0100 [TRACE] provider.terraform-provider-ovh_v2.10.0: Received downstream response: @caller=github.com/hashicorp/terraform-plugin-go@v0.26.0/tfprotov6/internal/tf6serverlogging/downstream_request.go:42 tf_req_duration_ms=5329 tf_req_id=b32225cb-d0ff-9f0d-b38a-f4be6b14386b @module=sdk.proto diagnostic_error_count=1 diagnostic_warning_count=0 tf_proto_version=6.8 tf_provider_addr=registry.terraform.io/ovh/ovh tf_resource_type=ovh_cloud_project_kube_oidc tf_rpc=ApplyResourceChange timestamp="2025-12-01T22:58:27.809+0100"
2025-12-01T22:58:27.811+0100 [ERROR] provider.terraform-provider-ovh_v2.10.0: Response contains error diagnostic: diagnostic_severity=ERROR diagnostic_summary="timeout while waiting kube 019adbb1-cfa5-76ab-9a00-ee2d784500d9 to be READY: unexpected state 'INSTALLING', wanted target 'READY'. last error: %!s(<nil>)" tf_provider_addr=registry.terraform.io/ovh/ovh tf_resource_type=ovh_cloud_project_kube_oidc tf_rpc=ApplyResourceChange @caller=github.com/hashicorp/terraform-plugin-go@v0.26.0/tfprotov6/internal/diag/diagnostics.go:58 @module=sdk.proto diagnostic_detail="" tf_proto_version=6.8 tf_req_id=b32225cb-d0ff-9f0d-b38a-f4be6b14386b timestamp="2025-12-01T22:58:27.811+0100"
2025-12-01T22:58:27.811+0100 [TRACE] provider.terraform-provider-ovh_v2.10.0: Served request: @module=sdk.proto tf_rpc=ApplyResourceChange @caller=github.com/hashicorp/terraform-plugin-go@v0.26.0/tfprotov6/tf6server/server.go:879 tf_proto_version=6.8 tf_provider_addr=registry.terraform.io/ovh/ovh tf_req_id=b32225cb-d0ff-9f0d-b38a-f4be6b14386b tf_resource_type=ovh_cloud_project_kube_oidc timestamp="2025-12-01T22:58:27.811+0100"
2025-12-01T22:58:27.812+0100 [TRACE] maybeTainted: module.cluster.ovh_cloud_project_kube_oidc.oidc encountered an error during creation, so it is now marked as tainted
2025-12-01T22:58:27.812+0100 [TRACE] terraform.contextPlugins: Schema for provider "registry.terraform.io/ovh/ovh" is in the global cache
2025-12-01T22:58:27.812+0100 [TRACE] NodeAbstractResouceInstance.writeResourceInstanceState to workingState for module.cluster.ovh_cloud_project_kube_oidc.oidc
2025-12-01T22:58:27.812+0100 [TRACE] NodeAbstractResouceInstance.writeResourceInstanceState: writing state object for module.cluster.ovh_cloud_project_kube_oidc.oidc
2025-12-01T22:58:27.812+0100 [TRACE] evalApplyProvisioners: module.cluster.ovh_cloud_project_kube_oidc.oidc is tainted, so skipping provisioning
2025-12-01T22:58:27.812+0100 [TRACE] maybeTainted: module.cluster.ovh_cloud_project_kube_oidc.oidc was already tainted, so nothing to do
2025-12-01T22:58:27.812+0100 [TRACE] terraform.contextPlugins: Schema for provider "registry.terraform.io/ovh/ovh" is in the global cache
2025-12-01T22:58:27.812+0100 [TRACE] NodeAbstractResouceInstance.writeResourceInstanceState to workingState for module.cluster.ovh_cloud_project_kube_oidc.oidc
2025-12-01T22:58:27.812+0100 [TRACE] NodeAbstractResouceInstance.writeResourceInstanceState: writing state object for module.cluster.ovh_cloud_project_kube_oidc.oidc
2025-12-01T22:58:27.813+0100 [DEBUG] State storage *remote.State declined to persist a state snapshot
2025-12-01T22:58:27.813+0100 [ERROR] vertex "module.cluster.ovh_cloud_project_kube_oidc.oidc" error: timeout while waiting kube 019adbb1-cfa5-76ab-9a00-ee2d784500d9 to be READY: unexpected state 'INSTALLING', wanted target 'READY'. last error: %!s(<nil>)
2025-12-01T22:58:27.813+0100 [TRACE] vertex "module.cluster.ovh_cloud_project_kube_oidc.oidc": visit complete, with errors
2025-12-01T22:58:27.813+0100 [TRACE] dag/walk: upstream of "module.cluster (close)" errored, so skipping
2025-12-01T22:58:27.813+0100 [TRACE] dag/walk: upstream of "module.cluster.provider[\"registry.terraform.io/ovh/ovh\"] (close)" errored, so skipping
2025-12-01T22:58:27.813+0100 [TRACE] dag/walk: upstream of "root" errored, so skipping
2025-12-01T22:58:27.813+0100 [DEBUG] states/remote: state read serial is: 14; serial is: 14
2025-12-01T22:58:27.813+0100 [DEBUG] states/remote: state read lineage is: e60c0344-f51f-f24d-8ded-fa2e19e48895; lineage is: e60c0344-f51f-f24d-8ded-fa2e19e48895
2025-12-01T22:58:27.817+0100 [INFO]  backend-s3: Uploading remote state: tf_backend.operation=Put tf_backend.req_id=69569f4c-04cd-0acd-dbaf-70a44fc74f4f tf_backend.s3.bucket=terraform-state tf_backend.s3.path=infrastructure/eidp-ovh-demo/terraform.tfstate
2025-12-01T22:58:27.823+0100 [DEBUG] backend-s3: HTTP Request Sent: aws.region=ams2 aws.s3.bucket=terraform-state aws.s3.key=infrastructure/eidp-ovh-demo/terraform.tfstate rpc.method=PutObject rpc.service=S3 rpc.system=aws-api tf_aws.custom_endpoint=true tf_aws.sdk=aws-sdk-go-v2 tf_backend.operation=Put tf_backend.req_id=69569f4c-04cd-0acd-dbaf-70a44fc74f4f tf_backend.s3.bucket=terraform-state tf_backend.s3.path=infrastructure/eidp-ovh-demo/terraform.tfstate net.peer.name=core.fuga.cloud http.request.header.authorization="AWS4-HMAC-SHA256 Credential=cbd7960bd96b4ffb8d68d0c743229c24/20251201/ams2/s3/aws4_request, SignedHeaders=accept-encoding;amz-sdk-invocation-id;amz-sdk-request;content-encoding;content-length;content-type;host;x-amz-content-sha256;x-amz-date;x-amz-decoded-content-length;x-amz-sdk-checksum-algorithm;x-amz-trailer, Signature=*****" http.request.header.x_amz_date=20251201T215827Z http.request.header.content_type=application/json http.request.header.content_encoding=aws-chunked http.request.header.x_amz_trailer=x-amz-checksum-sha256 http.request.header.accept_encoding=identity http.request.header.x_amz_content_sha256=STREAMING-UNSIGNED-PAYLOAD-TRAILER http.request.header.amz_sdk_invocation_id=3cbc1743-0cb7-40f5-82d4-7eb47512fb3c http.request.body="[Redacted: 66.1 KB (67,694 bytes), Type: application/json]" http.url=https://core.fuga.cloud:8080/terraform-state/infrastructure/eidp-ovh-demo/terraform.tfstate?x-id=PutObject net.peer.port=8080 http.user_agent="APN/1.0 HashiCorp/1.0 Terraform/1.14.0 (+https://www.terraform.io) aws-sdk-go-v2/1.39.2 ua/2.1 os/macos lang/go#1.25.4 md/GOOS#darwin md/GOARCH#arm64 api/s3#1.88.3 ft/s3-transfer m/G,Z,g" http.request.header.x_amz_sdk_checksum_algorithm=SHA256 http.method=PUT http.request_content_length=67694 http.request.header.x_amz_decoded_content_length=67612 http.request.header.amz_sdk_request="attempt=1; max=5"
2025-12-01T22:58:27.899+0100 [DEBUG] backend-s3: HTTP Response Received: aws.region=ams2 aws.s3.bucket=terraform-state aws.s3.key=infrastructure/eidp-ovh-demo/terraform.tfstate rpc.method=PutObject rpc.service=S3 rpc.system=aws-api tf_aws.custom_endpoint=true tf_aws.sdk=aws-sdk-go-v2 tf_backend.operation=Put tf_backend.req_id=69569f4c-04cd-0acd-dbaf-70a44fc74f4f tf_backend.s3.bucket=terraform-state tf_backend.s3.path=infrastructure/eidp-ovh-demo/terraform.tfstate http.response.body="" http.duration=75 http.status_code=200 http.response.header.x_amz_request_id=tx00000a785ed457523df9d-00692e0f83-313072377-default http.response.header.server="Ceph Object Gateway (squid)" http.response.header.date="Mon, 01 Dec 2025 21:58:27 GMT" http.response.header.etag="\"e45e4f16b0ca45c32ab4ec2f008b6ac3\"" http.response.header.accept_ranges=bytes
2025-12-01T22:58:27.903+0100 [DEBUG] provider.stdio: received EOF, stopping recv loop: err="rpc error: code = Unavailable desc = error reading from server: EOF"
2025-12-01T22:58:27.906+0100 [INFO]  provider: plugin process exited: plugin=.terraform/providers/registry.terraform.io/ovh/ovh/2.10.0/darwin_arm64/terraform-provider-ovh_v2.10.0 id=62917
2025-12-01T22:58:27.906+0100 [DEBUG] provider: plugin exited
```
</details>

### Panic Output

n/a

### Expected Behavior

The OVH Terraform provider should wait long enough to let the cluster get in the ready state again.

### Actual Behavior

TF fails to create the OIDC provider and the oidc resource is tainted according to TF. However, in the OVH console you can see that the OIDC provider is created.

### Steps to Reproduce

Please list the steps required to reproduce the issue, for example:
1. `terraform apply`

### References

I could not find any other related issues.

### Additional context

n/a

--- a/ovh/resource_cloud_project_kube_oidc.go
+++ b/ovh/resource_cloud_project_kube_oidc.go
@@ -126,7 +126,7 @@ func resourceCloudProjectKubeOIDCCreate(d *schema.ResourceData, meta interface{}
 	d.SetId(serviceName + "/" + kubeID)
 
 	log.Printf("[DEBUG] Waiting for kube %s to be READY", kubeID)
-	err = waitForCloudProjectKubeReady(config.OVHClient, serviceName, kubeID, []string{"REDEPLOYING"}, []string{"READY"}, d.Timeout(schema.TimeoutCreate))
+	err = waitForCloudProjectKubeReady(config.OVHClient, serviceName, kubeID, []string{"INSTALLING", "REDEPLOYING"}, []string{"READY"}, d.Timeout(schema.TimeoutCreate))
 	if err != nil {
 		return fmt.Errorf("timeout while waiting kube %s to be READY: %w", kubeID, err)
 	}
@@ -178,7 +178,7 @@ func resourceCloudProjectKubeOIDCUpdate(d *schema.ResourceData, meta interface{}
 	}
 
 	log.Printf("[DEBUG] Waiting for kube %s to be READY", kubeID)
-	err = waitForCloudProjectKubeReady(config.OVHClient, serviceName, kubeID, []string{"REDEPLOYING"}, []string{"READY"}, d.Timeout(schema.TimeoutUpdate))
+	err = waitForCloudProjectKubeReady(config.OVHClient, serviceName, kubeID, []string{"INSTALLING", "REDEPLOYING"}, []string{"READY"}, d.Timeout(schema.TimeoutUpdate))
 	if err != nil {
 		return fmt.Errorf("timeout while waiting kube %s to be READY: %w", kubeID, err)
 	}

